crates/uv-cli/src/lib.rs

```diff
@@ -2360,9 +2360,18 @@ pub struct LockArgs {
 #[derive(Args)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct AddArgs {
-    /// The packages to add, as PEP 508 requirements (e.g., `ruff==0.5.0`).
-    #[arg(required = true)]
-    pub requirements: Vec<String>,
+    /// Install all listed packages.
+    #[arg(group = "sources")]
+    pub packages: Vec<String>,
+
+    /// Install all packages listed in the given `requirements.txt` files.
+    ///
+    /// If a `pyproject.toml`, `setup.py`, or `setup.cfg` file is provided, uv will
+    /// extract the requirements for the relevant project.
```

Comment:
Should we allow these files? These seem like a bit of a misuse. We don't allow them in `uv run` https://github.com/astral-sh/uv/blob/2822dde8cb24f1486cef59228c723da931690c30/crates/uv/src/commands/project/run.rs#L74-L92

Reply:
Supporting `setup.py` and `setup.cfg` files could be a helpful feature, making it easier for users to transition to the pyproject.toml standard.

Reply:
It's possible that this could be a killer feature because you could import Poetry projects this way... But it doesn't actually work as-is. I'm gonna remove it and look at it in a separate PR.